src/functions.js

```diff
@@ -109,16 +175,43 @@ function sales( cars ){
   //   'Ford': 20000,
   //   'Vauxhall': 15000
   // }
-}
+ /*  for(var i=0;i<cars.length;i++){
+    for(var key in cars[i]){ */
+/*       if(cars[i]['make']==='Ford') cars['price'] +=cars[i]['price'];
+      if(cars[i][key]==='Land Rover') cars['price'] +=cars[i]['price'];
+      if(cars[i][key]==='Toyota') cars['price'] +=cars[i]['price'];
+      if(cars[i][key]==='Honda') cars['price'] +=cars[i]['price']; */
+      var newObj={'Ford':0 , 'Land Rover':0 , 'Toyota':0, 'Honda':0};
+      for(var i=0;i<cars.length;i++){
+         for(var key in cars[i]){
+             if(cars[i][key]=='Ford') newObj['Ford'] +=cars[i]['price']
+             //.push(arr[i][key]) //arr[i][key]
+             if(cars[i][key] =='Land Rover') newObj['Land Rover'] +=cars[i]['price']
+             if(cars[i][key]=='Toyota') newObj['Toyota']+=cars[i]['price']
+             if(cars[i][key] =='Honda') newObj['Honda']+=cars[i]['price']
+         }
+      }
+return newObj;
+}//lloc
```

Comment:
Whilst this one works, give some thought to what would happen if the test array contained BMW, Mercedes and Fiat vehicles. Hard coding values into a function is rarely a good idea, as it limits their reuse, which is a functions' main purpose. I'd be happy to take you through a different (reusable) solution tomorrow.